src/main/java/org/numenta/nupic/network/Region.java

```diff
@@ -457,6 +457,9 @@ Region connect(Region inputRegion) {
             @Override public void onError(Throwable e) { e.printStackTrace(); }
             @SuppressWarnings("unchecked")
             @Override public void onNext(Inference i) {
+                // TODO: 21: This is where classifierInput is set. Need to change
+                // it to respect only fields user has specified for classification
+                // with the new Parameters KEY.
```

Comment:
It is a comment I left for myself. Looks like I forgot to remove it with the others. 

It didn't end up being something I needed to do. I simply let the `classifierInput` be created and stored as it was when I began this PR. What I did instead what just not actually classify the `classifierInput` for fields that haven't had a classifier specified for them.